desktop-cosmic/cosmic-applets/autobuild/defines

```diff
@@ -0,0 +1,5 @@
+PKGNAME=cosmic-applets
+PKGSEC=COSMIC
+PKGDEP="cosmic-panel dbus gcc-runtime libxkbcommon pulseaudio systemd"
+BUILDDEP="dbus pulseaudio libinput mesa systemd rustc llvm just"
```

Comment:
How is libinput a build-time dependency (same for all the rest, other than `rustc`, `llvm`, and `just`)?